packages/block-editor/src/components/block-popover/inbetween.js

```diff
@@ -91,66 +91,53 @@ function BlockPopoverInbetween( {
 		};
 	}, [ previousElement, nextElement, isVertical ] );
 
-	const getAnchorRect = useCallback( () => {
+	const popoverAnchor = useMemo( () => {
 		if ( ( ! previousElement && ! nextElement ) || ! isVisible ) {
-			return {};
+			return undefined;
```

Comment:
Change no. 1: return `undefined` instead of an empty object, since that wouldn't be a valid value for the `anchor` prop